engine/src/main/java/org/terasology/engine/core/PathManager.java

```diff
@@ -97,11 +98,11 @@ private static Path findInstallPath() {
             }
         }
 
-        System.err.println(
+        System.err.printf(
```

Comment:
Does PMD have a valid point here in asking to use a logger instead of plain write to console?

Reply:
PathManager sometimes does stuff like this I wouldn't do elsewhere because it's so early in the boot sequence. Do you use the logger while initializing the object that's responsible for providing `getLogPath`?

(maybe? like, it'll still print WARN and ERROR to the console, right? idk)

Reply:
I would hope that the logger setup works for whatever code we write "afterwards", including the path manager and all that - but I'm not sure about that either. 

No need to change this right now in this PR, but maybe we can keep it in mind/try it out in the future.